Dockerfile

```diff
@@ -2,13 +2,13 @@
 
 # THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
 #
-# Generated on 2025-01-15T11:22:30Z by kres d83fc27c.
+# Generated on 2025-02-10T12:11:08Z by kres 48878ff.
 
 ARG TOOLCHAIN
 
-FROM ghcr.io/siderolabs/ca-certificates:v1.9.0 AS image-ca-certificates
+FROM ghcr.io/siderolabs/ca-certificates:v1.10.0-alpha.0-35-g85f8901 AS image-ca-certificates
 
-FROM ghcr.io/siderolabs/fhs:v1.9.0 AS image-fhs
+FROM ghcr.io/siderolabs/fhs:v1.10.0-alpha.0-35-g85f8901 AS image-fhs
```

Comment:
this would affect container images built... just fyi

Reply:
yeah, but we run it standalone always, so perhaps that's okay. Even if ran not standalone but layered with something, then better match newer versions